src/handlers/int-test-support/helpers/mock-data/csv/index.ts

```diff
@@ -1,2 +1,22 @@
-export * from "./addFauxDataToTestCases";
-export * from "./objectsToCsv";
+import { augmentTestCases, TestCases } from "./addFauxDataToTestCases";
+import { objectsToCSV } from "./objectsToCsv";
+import { testCases } from "./transformCSV-to-event-test-data";
+
+export const mockIdpCsvData = (): { testCases: TestCases; csv: Buffer } => {
```

Comment:
Like I said above this is just what was happening in the `beforeAll`. I decided to have this function return `{ testCases: TestCases; csv: Buffer }` rather than just `import`ing `testCases` in the test because this made `testCases` feel more tightly coupled to the csv.